src/components/create/create.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text } from '@storybook/addon-knobs';
+import notes from './notes.md';
+import Create from './create.js';
+
+storiesOf('Create', module)
```

Comment:
We shouldn't see redundant prop type info:

![image](https://user-images.githubusercontent.com/26925206/53880417-da6cf300-4008-11e9-95b1-72334bf949d8.png)

Reply:
thanks, done